### ISSUE TITLE
Unable to change Virtual Display size. (similar to Issue #34)

### ISSUE THREAD

**Issue by sjbanner** [Microsoft/linux-vm-tools#47]
I just enabled enhanced mode on my Kubuntu 18.04 VM, and it looks like it's working fine. However I selected a screen-size that's a bit too big - it cuts off the bottom part of the VM display. I looked at issue #34 which looks the same, however I can't get the "Edit Session" menu. I have tried all the options in that issue and several not in that issue, and the menu item just doesn't show up - if I could get it, that would probably solve the problem.

**Comment by jterry75** [comment]
@sjbanner - To get the enhanced size dialog box you need to close and re-open VMConnect. This display is only shown the first time an enhanced session is made in any given VMConnect window.

**Comment by sjbanner** [comment]
Then the question would be how does one do that? I am going to guess that's shutting down the VM and closing the window (though I thought I tried that, so I will let you confirm). But from your reply, I did a bit of searching about and found an article with the powershell command 'vmconnect localhost "<VM Name here>" /edit' which did the trick, so I am happy.

This might be a good thing to add to the FAQ.
